tests/Doctrine/Tests/ORM/ConfigurationTest.php

```diff
@@ -131,6 +133,14 @@ public function testSetGetMetadataCacheImpl(): void
         $this->assertSame($queryCacheImpl, $this->configuration->getMetadataCacheImpl());
     }
 
+    public function testSetGetMetadataCache(): void
+    {
+        $this->assertNull($this->configuration->getMetadataCache());
+        $cache = $this->createStub(CacheItemPoolInterface::class);
```

Comment:
Make sure to extend `DoctrineTestCase` for `createStub` to be available on PHP 7.1